packages/hop-node/src/gasboost/GasBoostTransaction.ts

```diff
@@ -569,20 +569,28 @@ class GasBoostTransaction extends EventEmitter implements providers.TransactionR
   }
 
   private async handleInflightTx (item: InflightItem) {
-    if (item.boosted) {
-      return
+    if (this.shouldBoost(item)) {
+      return this.boost(item)
     }
-    if (!this.shouldBoost(item)) {
-      return
+    const isLatestItem = item === this.getLatestInflightItem()
+    const shouldRebroadcast = isLatestItem && this.maxGasPriceReached
+    if (shouldRebroadcast) {
+      try {
+        await this.rebroadcastLatestTx()
+      } catch (err) {
+        const { isAlreadyKnown } = this.parseErrorString(err.message)
+        if (!isAlreadyKnown) {
+          this.logger.error('rebroadcastLatestTx error:', err)
+        }
```

Comment:
It might make sense to add a log here for debugging purposes. If the mempool is not behaving as we are assuming (as we've seen through various errors on various chains), it would be nice to see what error is being thrown at the bonder.

Reply:
should log all errors [now](https://github.com/hop-protocol/hop/pull/436/commits/7e3c6438101db68fe041558578fc23b73acb8c31)